mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -12802,3 +12802,60 @@ resource "google_container_cluster" "primary" {
 }
   `, name, cgroupMode)
 }
+
+func TestAccContainerCluster_withEnterpriseConfig(t *testing.T) {
```

Comment:
Can you add a final step to this test that removes the `enterprise_config` block entirely to remove the feature? We often see issues arise from removing a block, so it would be good to test